pontoon/administration/forms.py

```diff
@@ -11,6 +11,10 @@ class Meta:
                   'info_brief', 'url', 'width',
                   'links', 'disabled')
 
+    def __init__(self, *args, **kwargs):
+        super(ProjectForm, self).__init__(*args, **kwargs)
+        self.fields['locales'].required = False
+
```

Comment:
I was pretty convinced that locales where optional some time ago that's why I fixed it this way. Ok, I know what to do.